packages/modules/file/src/services/file-module-service.ts

```diff
@@ -49,6 +51,27 @@ export default class FileModuleService implements FileTypes.IFileModuleService {
     return Array.isArray(data) ? result : result[0]
   }
 
+  getUploadFileUrls(
```

Comment:
I just need to add a simple integration test to test the wiring. I'll do that tomorrow